service/kernel/service_data_utils.h

```diff
@@ -25,6 +25,9 @@
 #define __SERVICE_DATA_UTILS_H__
 
 #include "service/kernel/service_defines.h"
+#include <climits>
+#include <cfloat>
+#include <limits>
```

Comment:
forgotten includes?

Reply:
This includes are important for correct building. Constants used in this header are not redefined.